torch/nn/modules/utils.py

```diff
@@ -32,3 +32,34 @@ def _list_with_default(out_size: List[int], defaults: List[int]) -> List[int]:
     if len(defaults) <= len(out_size):
         raise ValueError('Input dimension should be at least {}'.format(len(out_size) + 1))
     return [v if v is not None else d for v, d in zip(out_size, defaults[-len(out_size):])]
+
+
+def consume_prefix_in_state_dict_if_present(state_dict: Dict[str, Any], prefix: str):
+    r"""Strip the prefix in state_dict metadata, if any.
+
+    Args:
+        state_dict (OrderedDict): a state-dict to be loaded to the model.
+        prefix (str): prefix.
```

Comment:
Maybe document how this is useful for ddp?

Reply:
Added a note in the docstring.